src/overlays/actors/ovl_Obj_Jg_Gakki/z_obj_jg_gakki.h

```diff
@@ -7,8 +7,9 @@ struct ObjJgGakki;
 
 typedef struct ObjJgGakki {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x90];
-} ObjJgGakki; // size = 0x1D4
+    /* 0x144 */ SkelAnime skelAnime;
+    /* 0x188 */ ColliderCylinder colliderCylinder;
+} ObjJgGakki; /* size = 0x1D4 */
```

Comment:
```suggestion
} ObjJgGakki; // size = 0x1D4
```
You can leave this size as it was commented originally